### ISSUE TITLE
Theme doesn't work in standard terminal or iTerm2

### ISSUE THREAD

**Issue by Integralist** [arcticicestudio/nord-vim#237]
I'm trying to use this theme but it's showing completely broken in _both_ macOS terminal app and even iTerm2 🤔 

At first I thought it might be something in my vim configuration, and then my bash configuration, and then even the terminal theme itself but nothing I do gets the colours to display how they're supposed to.

Example below (although it suggests vim-material is used I've manually set `:colorscheme nord`...
<img width="964" alt="Screenshot 2020-12-15 at 14 19 58" src="https://user-images.githubusercontent.com/180050/102226715-a1052f00-3ee0-11eb-85a0-4f27d350cdf2.png">

Any ideas what I might have configured incorrectly?

Thanks.

**Comment by arcticicestudio** [comment]
Hi @Integralist 👋🏼 

In order to ensure the problem is related to the Nord theme, can your please try to reproduce it with a minimal and clean configuration. This can be done by creating new configuration file with any name (e.g. `testrc`) and run Vim with the following command: `vim -u testrc`
For me as a theme author it is not possible to know if the theme is the root cause of the problem when there are other attributes set in the configuration because each one might break the theme.

Here's an minimal test configuration you can use to try to reproduce the problem using [vim-plug][]:

```vim
call plug#begin(expand('~/.vim/plugged'))
Plug 'arcticicestudio/nord-vim'
call plug#end()

set nocompatible
if (has("termguicolors"))
  set termguicolors
endif

syntax enable
colorscheme nord
```

Also just to make sure we're not missing something:

1. Did you install the required terminal emulator theme like [Nord iTerm2][n-iterm2] or [Terminal.app][n-iterm2]?
2. Did you run the `:PlugInstall` Vim command after adding the `Plug 'arcticicestudio/nord-vim'` line?

[n-iterm2]: https://www.nordtheme.com/ports/iterm2
[n-t.app]: https://github.com/arcticicestudio/nord-terminal-app
[vim-plug]: https://github.com/junegunn/vim-plug

**Comment by Integralist** [comment]
Hi @arcticicestudio thanks for the info!

> Did you install the required terminal emulator theme like Nord iTerm2 or Terminal.app?

...I wasn't aware that I would need a special/specific nord terminal theme installed for the nord vim theme to work correct 😬 

But reading https://www.nordtheme.com/docs/ports/vim/installation I see you've explained why this is necessary.

I also am now reading through [the gist](https://gist.github.com/XVilka/8346728) that page links to, to hopefully learn more about these terminal issues.

In summary: I installed the Nord terminal theme and the Nord vim theme now works fine 🎉 

Thank you!
